src/db/jobs.rs

```diff
@@ -0,0 +1,134 @@
+//! The `jobs` table provides a way to have scheduled jobs
+use anyhow::{Context as _, Result};
+use chrono::{DateTime, Duration, FixedOffset};
+use postgres_types::{FromSql, ToSql};
+use serde::{Deserialize, Serialize};
+use tokio_postgres::Client as DbClient;
+use uuid::Uuid;
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Job {
+    pub id: Uuid,
+    pub name: String,
+    pub scheduled_at: DateTime<FixedOffset>,
+    pub frequency: Option<i32>,
```

Comment:
It seems like this should be a u32, i.e., we should never have a *negative* duration here.

Reply:
(But actually see my other comment; this field is probably just going to go away entirely).